registry/accessing-the-registry.adoc

```diff
@@ -50,7 +50,7 @@ include::modules/registry-viewing-logs.adoc[leveloffset=+1]
 include::modules/registry-accessing-metrics.adoc[leveloffset=+1]
 
 .Additional resources
-
+* For more information on allowing pods to reference images across projects, see xref:../openshift_images/managing_images/using-image-pull-secrets.adoc#images-allow-pods-to-reference-images-across-projects_using-image-pull-secrets[Allowing pods to reference images across projects].
```

Comment:
Do we want to link to https://github.com/openshift/openshift-docs/blob/master/modules/images-allow-pods-to-reference-images-across-projects.adoc instead?

Reply:
@sferich888 and I had a slack convo and decided that the best customer experience may be to just include this info within this TOC, rather than link.

Reply:
Actually just discussed this with @bergerhoffer to get a second doc op, and she said linking is probably best here since this is a bit unrelated from the other items in the Accessing Registry TOC. I am going to update to make the Additional Resources visible in the TOC, so that should help raise awareness.